target/linux/ramips/dts/F9K1109.dts

```diff
@@ -0,0 +1,177 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "rt3883.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "belkin,f9k1109", "ralink,rt3883-soc";
+	model = "Belkin F9K1109";
+
+	aliases {
+		led-boot = &led_status_amber;
+		led-failsafe = &led_status_amber;
+		led-running = &led_status_blue;
+		led-upgrade = &led_status_amber;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status_amber: internet_amber {
+			label = "f9k1109:amber:internet";
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+		};
+
+		led_status_blue: internet_blue {
+			label = "f9k1109:blue:internet";
+			gpios = <&gpio0 12 GPIO_ACTIVE_LOW>;
+		};
+
+		usb1 {
+			label = "f9k1109:green:usb1";
+			gpios = <&gpio0 9 GPIO_ACTIVE_LOW>;
+		};
+
+		usb2 {
+			label = "f9k1109:green:usb2";
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+		};
+
+		wps_amber {
+			label = "f9k1109:amber:wps";
+			gpios = <&gpio0 0 GPIO_ACTIVE_LOW>;
+		};
+
+		wps_blue {
+			label = "f9k1109:blue:wps";
+			gpios = <&gpio0 10 GPIO_ACTIVE_LOW>;
+		};
+
+	};
+
+    gpio-keys-polled {
```

Comment:
@kapieyow Looks like you've some whitespace issue here. While at it, please can change this to generic name `keys` as suggested by [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist)
>  5. The name of a node should reflect the function of the device and not its model. Examples for generic node names can be found in Section 2.2.2 Generic Names Recommendation